src/rules/platform-agnostic.test.ts

```diff
@@ -179,4 +182,108 @@ describe('Platform Agnostic', () => {
 
   });
 
+  describe('rebase', () => {
```

Comment:
Maybe it's worth to have tests for cases when there occurrences of `>>>` and not `<<<` and vice versa